docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -1,25 +1,23 @@
 [[set-up-driver-plugin]]
-= Setting up the driver plugin
+= Setting Up the Driver Plug-in
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 1
 
 toc::[]
 
-[[hypervisor-drivers-overview]]
+[[setup-driver-plugin-overview]]
```

Comment:
Well, it is not required and I think the section you are referring to is misleading. Uniqueness is given by the full context. There would be no loss in information here, nor would there be an issue with other tooling. But let's move on